### ISSUE TITLE
Add "switch" component 

### ISSUE THREAD

**Issue by TixieSalander** [robinparisi/higgcss#14]
I would like add a "switch" component (something [like this](https://dribbble.com/shots/1034371-Day-Night-switch) based.
It should be "html radio button" based to be easy to use (without JS) and a11y compliant.

Also as it's a "hack" (input is masked and everything is in the `<label>`) I'll have to add a focus ring manually, so maybe we should add a styled outline ring feature through the framework first ?
